benchmarks/aes/compose_yamls/docker-compose-aes-nodejs.yaml

```diff
@@ -1,9 +1,8 @@
-version: "3.9"
+version: "3"
 services:
   ## NodeJS
   aes-nodejs:
     image: davidschall/aes-nodejs:latest
```

Comment:
It seemed you have now all images in the `vhiveease` repo which I really like! Thats good. You have also written a makefile to build the containers which is even better.
However, we should now change all the image tags to this new repo